src/js/service/LottoResult.js

```diff
@@ -73,7 +73,6 @@ export const getMyPrizeAmount = (lottoResult) => {
  * @returns {number}
  */
 export const getMyEarningRate = (purchasingAmount = 0, pirzeMoney = 0) => {
-  if ([purchasingAmount, pirzeMoney].every((n) => n === 0)) return 0;
   if (pirzeMoney - purchasingAmount < 0) {
     return -1 * 100;
   }
```

Comment:
```suggestion
  if (pirzeMoney < purchasingAmount) {
    return -100;
  }
```
이렇게 표현하는게 더 직관적으로 해석되는 것 같아요!